snapshot/pkg/volume/gluster/snapshot.go

```diff
@@ -60,9 +60,12 @@ func (h *glusterfsPlugin) SnapshotCreate(pv *v1.PersistentVolume, tags *map[stri
 	}
 
 	volumePath := spec.Glusterfs.Path
-	t := time.Now()
-	snapshotName := volumePath + "_" + t.Format(snapTimeFormat)
-
+	uuid := make([]byte, 16)
+	_, err := rand.Read(uuid)
```

Comment:
This looks rather wild... Wouldn't it be better to use "proper" UUID? We already have https://github.com/pborman/uuid among the dependencies anyway so I would definitely prefer using that.

Reply:
@tsmetana Sure.. Used `uuid` from referenced repo. Thanks.